packages/icons/src/img/sprite.svg

```diff
@@ -1207,6 +1207,13 @@
         clip-rule="evenodd"
       />
     </symbol>
+    <symbol viewBox="0 0 20 20" id="lp-icon-map-check">
+      <path
+        fill-rule="evenodd"
+        clip-rule="evenodd" 
+        d="M16.8943 0.695265C17.1154 0.83192 17.25 1.07332 17.25 1.33325V8.83325C17.25 9.24747 16.9142 9.58325 16.5 9.58325C16.0858 9.58325 15.75 9.24747 15.75 8.83325V2.54678L12.25 4.29678V10.4999C12.25 10.9141 11.9142 11.2499 11.5 11.2499C11.0858 11.2499 10.75 10.9141 10.75 10.4999V4.29678L7.25 2.54678V11.7031L8.50208 12.3291C8.87256 12.5143 9.02273 12.9648 8.83749 13.3353C8.65224 13.7058 8.20174 13.856 7.83126 13.6707L6.5 13.0051L1.83541 15.3374C1.60292 15.4537 1.32681 15.4412 1.1057 15.3046C0.884589 15.1679 0.75 14.9265 0.75 14.6666V3.83325C0.75 3.54917 0.910502 3.28948 1.16459 3.16243L6.16459 0.662432C6.37574 0.55686 6.62426 0.55686 6.83541 0.662432L11.5 2.99473L16.1646 0.662432C16.3971 0.546187 16.6732 0.55861 16.8943 0.695265ZM5.75 11.7031V2.54678L2.25 4.29678V13.4531L5.75 11.7031ZM13.1667 14.4393L15.9697 11.6363C16.2626 11.3434 16.7374 11.3434 17.0303 11.6363C17.3232 11.9291 17.3232 12.404 17.0303 12.6969L13.697 16.0303C13.5563 16.1709 13.3656 16.2499 13.1667 16.2499C12.9678 16.2499 12.777 16.1709 12.6363 16.0303L10.9697 14.3636C10.6768 14.0707 10.6768 13.5958 10.9697 13.3029C11.2626 13.01 11.7374 13.01 12.0303 13.3029L13.1667 14.4393Z"
```

Comment:
Seems like the path is drawn not in the center of the view box. I'll get you an updated path. 1 sec!

<img width="1046" alt="compared to other maps" src="https://github.com/launchdarkly/launchpad-ui/assets/895885/68e9b2b9-2863-4a24-a4ec-8faf21a0abc8">
<img width="215" alt="map not centered" src="https://github.com/launchdarkly/launchpad-ui/assets/895885/75ab4505-31d9-41b7-805c-f5399b8fd868">

Reply:
This should fix the alignment 😄 
```suggestion
        d="M17.894 2.695a.75.75 0 0 1 .356.638v7.5a.75.75 0 0 1-1.5 0V4.547l-3.5 1.75V12.5a.75.75 0 0 1-1.5 0V6.297l-3.5-1.75v9.156l1.252.626a.75.75 0 1 1-.67 1.342L7.5 15.005l-4.665 2.332a.75.75 0 0 1-1.085-.67V5.833a.75.75 0 0 1 .415-.67l5-2.5a.75.75 0 0 1 .67 0L12.5 4.994l4.665-2.333a.75.75 0 0 1 .73.033ZM6.75 13.703V4.547l-3.5 1.75v9.156l3.5-1.75Zm7.417 2.736 2.803-2.803a.75.75 0 1 1 1.06 1.061l-3.333 3.333a.75.75 0 0 1-1.06 0l-1.667-1.666a.75.75 0 1 1 1.06-1.061l1.137 1.136Z"
```